spec/controllers/ctc/questions/simplified_filing_income_offboarding_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Ctc::Questions::SimplifiedFilingIncomeOffboardingController do
   describe '#show' do
-    let(:intake) { create :ctc_intake, client: build(:client, tax_returns: [build(:tax_return, filing_status: filing_status)]) }
+    let(:intake) { create :ctc_intake, client: build(:client, tax_returns: [build(:tax_return, filing_status: filing_status, year: TaxReturn.current_tax_year)]) }
```

Comment:
Oh dear - what a subtle thing to change. Maybe it makes sense to change the factory. Not a blocker.

Reply:
l8r